packages/airnode-node/src/api/unsafe-evaluate.ts

```diff
@@ -1,13 +1,117 @@
+import assert from 'assert';
+import async_hooks from 'async_hooks';
+import buffer from 'buffer';
+import child_process from 'child_process';
+import cluster from 'cluster';
+import console from 'console';
+import constants from 'constants';
+import crypto from 'crypto';
+import dgram from 'dgram';
+import dns from 'dns';
+import events from 'events';
+import fs from 'fs';
+import http from 'http';
+import http2 from 'http2';
+import https from 'https';
+import inspector from 'inspector';
+import module from 'module';
+import net from 'net';
+import os from 'os';
+import path from 'path';
+import perf_hooks from 'perf_hooks';
+import process from 'process';
+import readline from 'readline';
+import repl from 'repl';
+import stream from 'stream';
+import string_decoder from 'string_decoder';
+import timers from 'timers';
+import tls from 'tls';
+import trace_events from 'trace_events';
+import tty from 'tty';
+import url from 'url';
+import util from 'util';
+import v8 from 'v8';
+import vm from 'vm';
+import worker_threads from 'worker_threads';
+import zlib from 'zlib';
+
+const buildInNodeModules = {
```

Comment:
Do we need to pass all of these? Will it work OK when user overrides the name of the built in module (e.g. does `const crypto = ...`)? I think it will work, but asking just in case.

Reply:
Many things don't work without this, I'm not entirely sure why, webpack seems to override the default require function and strips out everything else. `fs` definitely doesn't work. I just got a list of default modules for node and included most of the modules except the deprecated ones. I did this because I don't know what we might need in future and the cost of including these extra modules seems negligible.